bpf_linux.go

```diff
@@ -39,7 +39,7 @@ func loadSimpleBpf(progType BpfProgType, ret uint32) (int, error) {
 	attr := BPFAttr{
 		ProgType: uint32(progType),
 		InsnCnt:  uint32(len(insns)),
-		Insns:    uintptr(unsafe.Pointer(&insns)),
+		Insns:    uintptr(unsafe.Pointer(&insns[0])),
```

Comment:
I think would be cleaner to remove the second instruction (line 36), the result should be the same

Reply:
See the bug.